src/Sfx-Proxy/proxy.js

```diff
@@ -9,6 +9,9 @@ const path = require('path');
 let recordRequest = process.argv.includes("-r");
 let replayRequest = process.argv.includes("-p");
 
+console.log("record requests : " + recordRequest);
```

Comment:
How noisy will this be?

Reply:
it only displays once when you start the script. It felt like a good idea to make sure the flags passed in are correct and made it easier to check when it started up.